basket/news/forms.py

```diff
@@ -78,6 +78,10 @@ class CommonVoiceForm(forms.Form):
     first_contribution_date = forms.CharField(required=False,
                                               max_length=20,
                                               validators=[validate_datetime_str])
+    last_active_date = forms.CharField(required=False,
+                                       max_length=20,
+                                       validators=[validate_datetime_str])
+    two_day_streak = forms.NullBooleanField()
```

Comment:
Do you think it's worth testing this value as `Null`/an empty string?

Reply:
The test that doesn't include that value is kindof a test for that. It's empty, which means that field will be `None` which is then filtered out and not sent to the task at all. I think that should cover pretty well. Could possibly be better, but this view is protected by API key so we can better expect good values from our partner on this one as well.